src/components/formik/devicePicker/index.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
```

Comment:
Code base is using kebab case for file names. Please use this convention here too.